src/glob.zig

```diff
@@ -109,8 +110,26 @@ const CursorState = struct {
     }
 };
 
+const log = bun.Output.scoped(.glob, false);
+
 pub const BunGlobWalker = GlobWalker_(null, false);
 
+fn getcwd(buf: *[bun.MAX_PATH_BYTES]u8) Maybe([]const u8) {
+    if (bun.Environment.isWindows) {
+        var wbuf: bun.WPathBuffer = undefined;
+        const ptr = bun.windows._wgetcwd(&wbuf, wbuf.len) orelse {
```

Comment:
is using c-runtime required for this? can [GetCurrentDirectory](https://learn.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-getcurrentdirectory) or `bun.getcwd` be used instead?

actually i see that wgetcwd is using wide chars, while the windows api is ansi. hmm.

Reply:
switched to using fs.top_level_dir instead